aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -112,11 +113,16 @@ impl AptosVMImpl {
         };
 
         let features = Features::fetch_config(&storage).unwrap_or_default();
+
+        // If no chain ID is in storage, we assume we are in a testing environment and use ChainId::TESTING
+        let chain_id = ChainId::fetch_config(&storage).unwrap_or_else(ChainId::test);
```

Comment:
Do we have any cases where ChainId is not set?

Reply:
Yes, in our testing environments, like when running `cargo test` in `aptos-move/framework`